Dockerfile

```diff
@@ -1,14 +1,14 @@
-# FROM python:3.9-slim
-FROM python@sha256:c5f60863db103c951595f110def9244c1e09efe9e8d072cfac3da39310bc8cc8
+FROM python:latest
+# FROM python@sha256:c5f60863db103c951595f110def9244c1e09efe9e8d072cfac3da39310bc8cc8
 
 # install additional OS packages.
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && apt-get -y install --no-install-recommends zip unzip curl postgresql-client build-essential jq
 
 # Install Geosupport
-ARG RELEASE=21c
+ARG RELEASE=21a
 ARG MAJOR=21
-ARG MINOR=3
```

Comment:
it is not, it's the most recent I could get to work though. I think I remember how to get to the newest version, let me try that out

Reply:
I mention it because I thought we went through how to get the latest version of geosupport with Amanda - I think you have to sign up via email and that the way the version is written is different (if i remember correctly)

Reply:
[this action](https://github.com/NYCPlanning/db-facilities/actions/runs/2373504086) builds with the newest version of geosupport, hopefully it will work